partcad/src/partcad/part_factory_step.py

```
@@ -136,22 +139,25 @@
-                for i in range(reader.NbRootsForTransfer()):
-                    reader.TransferRoot(i + 1)
-
-                builder = TopoDS_Builder()
-                compound = TopoDS_Compound()
-                builder.MakeCompound(compound)
-                for i in range(reader.NbShapes()):
-                    occ_shape = reader.Shape(i + 1)
-                    builder.Add(compound, occ_shape)
-                shape = compound
+                with pc_tracer.start_as_current_span(
+                    "*PartFactoryStep.instantiate.{OCP.STEPControl.STEPControl_Reader}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)